packages/core/src/components/footer/test/default/footer.e2e.ts

```diff
@@ -27,7 +27,7 @@ test.describe('tds-footer-default', () => {
 
   test('Footer contains copyright text', async ({ page }) => {
     await page.goto(componentTestPath);
-    const footerCopyrightText = page.getByText('Copyright © 2023 Scania');
+    const footerCopyrightText = page.getByText('Copyright © 2024 Scania');
```

Comment:
```suggestion
    const footerCopyrightText = page.getByText(`Copyright © ${new Date().getFullYear()}  Scania`);
```

Not sure if this works in the testing environment, but would be nice to not have to update this every year.

Reply:
Does work Sir, I just removed extra space ;) Thanks for your help man!